build/azure-pipelines/darwin/sql-product-build-darwin-signing.yml

```diff
@@ -0,0 +1,82 @@
+steps:
+  - task: InstallAppleCertificate@2
```

Comment:
This won't work - the task in the other darwin file needs this apple certificate installed.

Reply:
Nevermind its repeated not moved.

Reply:
I didn't delete this task from the existing macOS job, it is still there.